import-export-cli/integration/testutils/throttlePolicy_testUtils.go

```diff
@@ -0,0 +1,167 @@
+	policy := client.GetThrottlePolicy(uuid, policyType)
+	client.DeleteThrottlePolicy(uuid, policyType)
+	return ThrottlePolicyStructToMap(policy)
+}
+
+// Validates whether throttling policies are equal
+func ValidatePoliciesEqual(t *testing.T, args *ThrottlePolicyImportExportTestArgs, importedPolicy map[string]interface{}) {
+	exportedPolicy := args.Policy
+	exportedPolicy[PolicyIDKey] = PolicyIDReset
+	importedPolicy[PolicyIDKey] = PolicyIDReset
+	assert.Equal(t, exportedPolicy, importedPolicy)
+}
+
+// Converts Throttling policy struct to map
+func ThrottlePolicyStructToMap(policy interface{}) (map[string]interface{}, error) {
+	var throttlePolicy map[string]interface{}
+	marshalled, _ := json.Marshal(policy)
+	err := json.Unmarshal(marshalled, &throttlePolicy)
+	return throttlePolicy, err
+}
+
+// Cleanup func for exported throttling policy file
+func RemoveExportedThrottlingPolicyFile(t *testing.T, file string) {
+	t.Log("base.RemoveExportedThrottlingPolicyFile() - file path:", file)
+	if _, err := os.Stat(file); err == nil {
+		err := os.Remove(file)
+		if err != nil {
+			t.Fatal(err)
+		}
+	}
+}
+
+// Validates whether the throttling policy list is complete
+func ValidateThrottlePoliciesList(t *testing.T, args *ThrottlePolicyImportExportTestArgs) {
+	t.Helper()
+
+	// Setup apictl envs
+	base.SetupEnv(t, args.SrcAPIM.GetEnvName(), args.SrcAPIM.GetApimURL(), args.SrcAPIM.GetTokenURL())
+
+	base.Login(t, args.SrcAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+
+	base.WaitForIndexing()
+
+	output, _ := listThrottlePolicies(t, args)
+
+	throttlePoliciesList := args.SrcAPIM.GetThrottlePolicies(t, args.CtlUser.Username, args.CtlUser.Password)
+
+	ValidateListThrottlePoliciesEqual(t, output, throttlePoliciesList)
+}
+
+// Validates whether the throttling policy list with Json Pretty Format is complete
+func ValidateThrottlePoliciesListWithJsonPrettyFormat(t *testing.T, args *ThrottlePolicyImportExportTestArgs) {
+	t.Helper()
+
+	// Setup apictl envs
+	base.SetupEnv(t, args.SrcAPIM.GetEnvName(), args.SrcAPIM.GetApimURL(), args.SrcAPIM.GetTokenURL())
+
+	base.Login(t, args.SrcAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+
+	base.WaitForIndexing()
+
+	output, _ := listThrottlePoliciesWithJsonPrettyFormat(t, args)
+
+	throttlePoliciesList := args.SrcAPIM.GetThrottlePolicies(t, args.CtlUser.Username, args.CtlUser.Password)
+
+	ValidateListThrottlePoliciesEqual(t, output, throttlePoliciesList)
+}
+
+// get the Throttle policy list apictl output
+func listThrottlePolicies(t *testing.T, args *ThrottlePolicyImportExportTestArgs) (string, error) {
+	output, err := base.Execute(t, "get", "policies", "rate-limiting", "-e", args.SrcAPIM.EnvName, "-k", "--verbose")
+	return output, err
+}
+
+// get the Throttle policy list apictl output in Json Pretty format
+func listThrottlePoliciesWithJsonPrettyFormat(t *testing.T, args *ThrottlePolicyImportExportTestArgs) (string, error) {
```

Comment:
Can't we reuse the same function named _listThrottlePolicies_ for this purpose as well?
We can add an if condition to check a boolean value. If the boolean value is true, we can append `"--format", "\"{{ jsonPretty . }}\""` to the executing command.

Let's try to reuse the same functions as much as possible.